corehq/apps/export/views/utils.py

```diff
@@ -215,8 +216,12 @@ def terminology(self):
 
     def create_new_export_instance(self, schema):
         instance = super(ODataFeedMixin, self).create_new_export_instance(schema)
+        settings = get_default_export_settings_for_domain(schema.domain)
         instance.is_odata_config = True
         instance.transform_dates = False
+        if hasattr(instance, 'include_errors'):
+            instance.include_errors = settings.odata_include_duplicates
+        instance.split_multiselects = settings.odata_expand_checkbox
```

Comment:
I don't particularly like checking if the class has that attribute, but only one of the `ExportInstance` subclasses has this attribute so I decided to seize the opportunity to write this logic once instead of in the 2 subclasses.

Though looking at this again, it might be better to add this logic directly to the subclasses here:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/views/new.py#L314-L333

Reply:
agreed